dcos.yml

```diff
@@ -10,38 +10,67 @@
         timeout: 120
 
 - name: Collect DC/OS versions
-  hosts: all
+  hosts: all!win_agents
+  gather_facts: no
   tasks:
     - name: Collect DC/OS versions
       setup:
 
+
 - name: DC/OS Requirements
-  hosts: all
+  hosts: all!win_agents
+  gather_facts: no
```

Comment:
please remove this line. We should still gather facts here

Reply:
agreed, removed